ENIGMAsystem/SHELL/Universal_System/Extensions/FileDropper/About.ey

```diff
@@ -0,0 +1,13 @@
+%e-yaml
+---
+
+Name: File Dropper
+Identifier: FileDroper
```

Comment:
Typo here, that's why the AppVeyor job failed. Just tweak it and it should be passing.

Reply:
Alright, I fixed it, and all checks are passing.